Bump version:

HougeLangley opened the pull request:
app-crypt/archinux-keyring: bump to 20221220
app-dicts/fcitx5-pinyin-moegirl: bump to 20221214
app-editors/appflowy-bin: bump to 0.0.9
app-editors/typoa: bump to 1.4.7
app-i18n/fcitx-lua: bump to 5.0.10
app-misc/bcompare: bump to 4.4.4.27058
app-misc/gpick: bump to 0.3
app-misc/yq-bin: bump to 4.30.6
app-mobilephone/scrcpy: bump to 1.25

close: #2621
close: #1835
close: #2083
close: #2054
close: #2626

HougeLangley committed:
Bump version:

app-crypt/archinux-keyring: bump to 20221220
app-dicts/fcitx5-pinyin-moegirl: bump to 20221214
app-editors/appflowy-bin: bump to 0.0.9
app-editors/typoa: bump to 1.4.7
app-i18n/fcitx-lua: bump to 5.0.10
app-misc/bcompare: bump to 4.4.4.27058
app-misc/gpick: bump to 0.3
app-misc/yq-bin: bump to 4.30.6
app-mobilephone/scrcpy: bump to 1.25

Universebenzene commented on app-crypt/archlinux-keyring/archlinux-keyring-20221220.ebuild:
```diff
@@ -0,0 +1,34 @@
+BDEPEND="app-crypt/sequoia-sq"
+
+src_prepare() {
+	default
```
这写出来不是画蛇添足么

Universebenzene commented on app-editors/typora/typora-1.4.7.ebuild:
```diff
@@ -0,0 +1,43 @@
+	mv "${S}/usr" "${D}"
+
+	pushd "${D}/usr/share/doc" > /dev/null || die
+	mv ${PN} ${P}
```
`pushd` 和 `popd` 没落 `|| die` ，`mv` 倒都落了

HougeLangley replied:
@Universebenzene 天苯的意思是 `mv xxxx || die` 这种吗？

Universebenzene replied:
@HougeLangley 对

Universebenzene commented on app-misc/bcompare/bcompare-4.4.4.27058.ebuild:
```diff
@@ -0,0 +1,40 @@
+	sed -i ./install.sh -e 's/^# Create desktop entry and place.*/exit 0/g'
+	sed -i ./install.sh -e "s%^# Copy the files.*%BC_BIN=\"$D/\$BC_BIN\"; BC_LIB=\"$D/\$BC_LIB\";%g"
+	sed -i ./install.sh -e "s/^\texit 1.*//g"
+	./install.sh --prefix="${targetdir}"
```
四句都落 `|| die`

HougeLangley replied:
末尾加上` || die` ？

Universebenzene replied:
这些都是裸shell命令，应该都得带上 `|| die`

Universebenzene commented on app-misc/bcompare/bcompare-4.4.4.27058.ebuild:
```diff
@@ -0,0 +1,40 @@
+"
+
+src_unpack() {
+	unpack ${A}
```
这个和默认行为有什么不一样么

HougeLangley replied:
直接干掉这行？

Universebenzene replied:
我感觉这和src_unpack的默认行为是一样的，按说是直接把这段删了就行，可能需要测试下

Universebenzene commented on app-crypt/archlinux-keyring/archlinux-keyring-20221220.ebuild:
```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2021 Gentoo Authors
```
header年份该改了

Universebenzene commented on app-editors/appflowy-bin/appflowy-bin-0.0.9.ebuild:
```diff
@@ -0,0 +1,55 @@
+# Copyright 2022 Gentoo Authors
```
header年份该改了

Universebenzene commented on app-editors/typora/typora-1.4.7.ebuild:
```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2022 Gentoo Authors
```
header年份该改了

Universebenzene commented on app-i18n/fcitx-lua/fcitx-lua-5.0.10.ebuild:
```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2022 Gentoo Authors
```
header年份该改了

Universebenzene commented on app-misc/bcompare/bcompare-4.4.4.27058.ebuild:
```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2021 Gentoo Authors
```
header年份该改了

Universebenzene commented on app-misc/gpick/gpick-0.3.ebuild:
```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2022 Gentoo Authors
```
header年份该改了

Universebenzene commented on app-misc/yq-bin/yq-bin-4.30.6.ebuild:
```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2022 Gentoo Authors
```
header年份该改了

Universebenzene commented on app-editors/typora/typora-1.4.7.ebuild:
```diff
@@ -0,0 +1,43 @@
+}
+
+pkg_postinst() {
+	update-desktop-database
```
这句原来是裸shell命令，为啥不用xdg相关的eclass呢

HougeLangley replied:
@Universebenzene 天苯的意思是改成：inherit 添加 xdg 这个部分就不要了？

Universebenzene replied:
@HougeLangley 对，加xdg，去desktop，再去掉pkg_postinst

HougeLangley replied:
@Universebenzene files整个文件夹都可以不要了吧，我觉得

HougeLangley committed:
Fixed ebuild header to 2023

app-arch/zchunk
app-crypt/archlinux-keyring
app-dicts/fcitx5-pinyin-moegirl
app-editors/appflowy-bin
app-editors/typora
app-i18n/fcitx-lua
app-misc/bcompare
app-misc/gpick
app-misc/yq-bin
app-mobilephone/scrcpy
sys-apps/zypper
sys-kernel/xanmod-rt
sys-libs/libsolv
sys-libs/libsolv
sys-libs/libzypp
sys-libs/libzypp

HougeLangley committed:
Fixed headers upstream and mistakes

HougeLangley committed:
Fixed:

sys-kernel/xanmod-rt !sys-kernel/xanmod-edge->!sys-kernel/xanmod-kernel

HougeLangley committed:
Fixed:

1. app-editors/typora: drop old ebuild
2. app-editors/typora/files: drop
3. app-misc/becompare: drop old ebuild

HougeLangley committed:
Fixed:

1. app-misc/gpick: WhitespaceFound line: 10
2. app-editors/typora: restora files
3. app-editors/typora: delete desktop eclass

HougeLangley committed:
Fixed:

app-mobilephone: delete ninja-utils
